onnxruntime/test/onnx/main.cc

```diff
@@ -299,8 +311,13 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"BatchNorm2d_momentum_eval", "disable reason"},
       {"BatchNorm3d_eval", "disable reason"},
       {"BatchNorm3d_momentum_eval", "disable reason"},
+#if defined(__GNUG__) && !defined(__LP64__)
       {"constantofshape_float_ones", "test data bug"},
       {"constantofshape_int_zeros", "test data bug"},
+#else
+      {"constantofshape_float_ones", "test data bug", {9,10}},
+      {"constantofshape_int_zeros", "test data bug", {9,10}},
```

Comment:
>{9,10}}, [](start = 53, length = 8)

I have a question here. We normally run against versions 1.4.1, 1.5.0 and the tip of the master. Does this enable tip of the master to run?

Reply:
maybe we can have a flag `is_fixed_in_master` and while running against tip of onnx master, the test runner should expect the test to pass if `is_fixed_in_master` is set to `true` (even if the opset version of the op is found in the broken versions list ?

Reply:
Thanks for the ingenious idea!
As discussed offline, let's go with commit id.